iterators.tex

```diff
@@ -588,7 +589,7 @@
   template <class I>
   concept bool WeakIterator@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ WeaklyIncrementable<I>@\newtxt{()}@ &&
-      requires(I i) {
+      requires(@\newtxt{const }@I i) {
         { *i } -> auto&&; // pre: i is dereferenceable
```

Comment:
I know this is unrelated to your changes, but this comment hanging out here in space sticks out now. I'll need to do some digging to see if it's even necessary.